lib/logstash/codecs/fluent.rb

```diff
@@ -67,6 +82,15 @@ def forwardable_tag(event)
 
   private
 
+  def decode_fluent_time(fluent_time)
+    case fluent_time
+    when Fixnum
+      fluent_time
+    when EventTime
+      Time.at(fluent_time.sec, fluent_time.nsec)
```

Comment:
We are using the latest version of this plugin and are running into a problem, which in our opinion is caused by this line here. As far as I can see, calling `Time.at` with just two parameters, expects the second parameter to specify microseconds and not nanoseconds, which in our case changed the timestamps and suddenly added roughly 10 minutes.

Shouldn't this read `Time.at(fluent_time.sec, fluent_time.nsec / 1000.0)` instead?